Resources/Prototypes/DeltaV/Catalog/uplink_catalog.yml

```diff
@@ -43,7 +43,7 @@
   discountDownTo:
     Telecrystal: 2
```

Comment:
should probably make discount 3 tc too

Reply:
Ah, thought the discount could be in range from 6-2 not that it would always discount to 2. Will fix when I get home